account_invoice_refund_reinvoice/models/account_move.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+
+from odoo.addons.account.models.account_move import TYPE_REVERSE_MAP
+
+TYPE_REVERSE_MAP.update({"out_refund": "out_invoice", "in_refund": "in_invoice"})
+
+
+class AccountMove(models.Model):
+
+    _inherit = "account.move"
+
+    def _prepare_reinvoice_reversal(self):
+        return {
+            "ref": _("Reversal of: %s") % (self.name),
+            "journal_id": self.journal_id.id,
+            "invoice_payment_term_id": None,
+            "invoice_user_id": self.invoice_user_id.id,
+        }
+
+    def _reverse_move_vals(self, default_values, cancel=True):
```

Comment:
I think that this function was removed on 16. Also, I think you found another way to do the job.

Reply:
You're right! I removed the unnecessary overridden method.